agent/internal/client/methods.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/evergreen-ci/evergreen/model"
 	"github.com/evergreen-ci/evergreen/model/artifact"
 	"github.com/evergreen-ci/evergreen/model/manifest"
+	"github.com/evergreen-ci/evergreen/model/patch"
```

Comment:
here too: the patch package is aliased on the next line as `patchmodel`. 
I don't know why we aliased it, but we should probably just import it once either with or without the alias.